docs/Documentation/Applications/openfoam.md

```diff
@@ -50,3 +50,71 @@ In order to build OpenFOAM with cray-mpich, two files need to be edited.
 Before you install OpenFOAM, make sure to load `Prgenv-gnu`.
 This will load gcc and cray-mpich. 
 Make sure the same module is loaded at runtime.
+
+<!-- ## OpenFOAM on Kestrel -->
+
+## Running OpenFOAM cases using pre-installed modules
+
+There are several modules for builds of OpenFOAM. After logging in to a CPU node on the Kestrel, please use the `module avail` command to view available versions. 
+
+```
+CPU $ module avail openfoam
+----------------------------- /nopt/nrel/apps/cpu_stack/modules/default/application -----------------------------
+   openfoam/v2306-openmpi-gcc      openfoam/9-craympich (D)    openfoam/11-craympich
+   openfoam/v2406-craympich-gcc    openfoam/9-ompi             openfoam/12-intelmpi
+```
+
+**We encourage users to switch to newer versions and perform strong & weak scaling [tests](https://hpc-wiki.info/hpc/Scaling) for their setup when submitting a new large job. According to some user reports, the application has shown to scale very poorly beyond 2 nodes.**
+
+??? example "Sample job script: Kestrel"
+
+
+    ```
+    #!/bin/bash
+    #SBATCH --job-name=myOpenFOAMjob
+    #SBATCH --account=<your-account-name>
+    #SBATCH --output=foamOutputLog.out
+    #SBATCH --error=foamErrorLog.out
+    #SBATCH --mail-user=<yourEmailAddress>@nrel.gov 
+    #SBATCH --nodes=2
+    #SBATCH --ntasks-per-node=104 # set number of MPI ranks per node
+    #SBATCH --cpus-per-task=1 # set number of OpenMP threads per MPI rank
+    #SBATCH --time=04:00:00
+    
+    
+    module load openfoam/v2306-openmpi-gcc
```

Comment:
Do you recommend that users use the openmpi build of openfoam, or this just an example? Could you explain that a bit above?